source/core/lethe_grid_tools.cc

```diff
@@ -1511,182 +1511,214 @@ LetheGridTools::find_point_triangle_distance(
   const std::vector<Point<dim>> &triangle,
   const Point<dim>              &point)
 {
+  double D;
+
   const Point<dim> &point_0 = triangle[0];
   const Point<dim> &point_1 = triangle[1];
```

Comment:
Ohh ok now I understand, you made it work in 2D also. Nice ok.
This was not clear to me
Maybe D could be distance? Would be maybe a bit easier.